Tests/SonarScanner.MSBuild.PreProcessor.Test/CacheProcessorTests.cs

```diff
@@ -97,9 +101,15 @@ private static string CreateFile(string root, string fileName, string content, E
         private static string Serialize(byte[] value) =>
             string.Concat(value.Select(x => x.ToString("x2")));
 
-        private static ProcessedArgs CreateProcessedArgs()
+        private static ProcessedArgs CreateProcessedArgs(string projectKey = "key", string pullRequestBase = null)
```

Comment:
The remaining changes can be reverted too (it would cause a rebase conflict for me otherwise)